### DISCUSSION TITLE
Vector version of the PostgREST logo

### DISCUSSION THREAD

**Discussion by terryburton** [PostgREST/postgrest#2513] [Q&A] [answered]
Hi,

Is there a vector version of the PostgREST logo available? Any of SVG, EPS, AI formats would be ideal for implementation documentation purposes.

Thanks.

**Comment by steve-chavez** [comment]
Hello,

The PostgREST logo was created before my time as maintainer so unfortunately I don't have it :disappointed:

Pinging @casalaina and @begriffs just in case.

**Reply by terryburton** [reply]
@steve-chavez Thank you.

**Comment by casalaina** [comment, marked as answer]
Oh wild, blast from the past here. Didn't know this project kept going! Found it on an old hard drive, here it is attached as SVG and AI!
[postgREST logo.zip](https://github.com/PostgREST/postgrest/files/9800065/postgREST.logo.zip)

**Reply by terryburton** [reply]
Thanks. Very much appreciated.
